File: problemas/ad-hoc/1794/1794.ml
Scanf.scanf "%d\n" (fun n ->
    Scanf.scanf "%d %d\n" (fun la lb ->
        Scanf.scanf "%d %d\n" (fun sa sb ->
            if la <= n && n <= lb && sa <= n && n <= sb then
              Printf.printf "possivel\n"
            else Printf.printf "impossivel\n")))

File: problemas/ad-hoc/1794/1794.go
package main

import (
    "fmt"
)

func main() {
    var n, la, lb, sa, sb int

    fmt.Scanf("%d", &n)
    fmt.Scanf("%d %d", &la, &lb)
    fmt.Scanf("%d %d", &sb, &sb)

    if la <= n && n <= lb && sa <= n && n <= sb {
        fmt.Printf("possivel\n")
    } else {
        fmt.Printf("impossivel\n")
    }
}
